src/win/fs.c

```diff
@@ -113,9 +113,44 @@ const WCHAR LONG_PATH_PREFIX_LEN = 4;
 const WCHAR UNC_PATH_PREFIX[] = L"\\\\?\\UNC\\";
 const WCHAR UNC_PATH_PREFIX_LEN = 8;
 
+static DWORD uv__file_symlink_flags;
+
+static int uv__usermode_symlink_available(void) {
```

Comment:
We would get `ERROR_INVALID_PARAMETER`. I guess we could then try to retry the call without this flag.

In any case, I think that testing for the feature before using it is nicer.